packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -1,10 +1,23 @@
 import React, { PropTypes } from 'react';
-import VirtualizedList from '../../VirtualizedList';
+import VirtualizedList, { SORT_BY } from '../../VirtualizedList';
 import CellTitle from '../../VirtualizedList/CellTitle';
 import CellActions from '../../VirtualizedList/CellActions';
 
+function adaptOnSort(onChange) {
+	if (!onChange) {
+		return null;
+	}
+	return function onSortChange({ sortBy, sortDirection }) {
+		return onChange(
+			null,
```

Comment:
:/ we don't have it. It's react-virtualized sort on click callback that provides only those infos